dev/Materials/Acrylic/AcrylicBrush_rs2_themeresources.xaml

```diff
@@ -217,7 +217,7 @@
             <SolidColorBrush x:Key="SystemControlTransientBorderBrush" Color="#000000" Opacity="0.36" />
             <local:AcrylicBrush
                 x:Key="AcrylicBackgroundFillColorDefaultBrush"
-                TintColor="#1C1C1C"
+                TintColor="#2C2C2C"
                 TintOpacity="0.8"
                 FallbackColor="#2C2C2C"
                 BackgroundSource="HostBackdrop"/>
```

Comment:
TintColor="#2C2C2C" for AcrylicInAppFillColorDefaultBrush

Reply:
Updated now.